src/components/organisms/NavBarSchedule/NavBarSchedule.tsx

```diff
@@ -97,7 +113,9 @@ export const NavBarSchedule: React.FC<NavBarScheduleProps> = ({
 
   const [selectedDayIndex, setSelectedDayIndex] = useState(0);
 
-  const weekdays = useMemo(() => {
+  // TODO: extract a new useMemo'd  const startOfSelectedDay = addDays(firstDayOfSchedule, selectedDayIndex);
+  //  Then reuse that in all the places that are currently duplicating this logic
```

Comment:
[TODO] extract a new `useMemo`'d  `const startOfSelectedDay = addDays(firstDayOfSchedule, selectedDayIndex);` then reuse that in all the places that are currently duplicating this logic